setup.py

```diff
@@ -68,3 +68,16 @@
               'bubblegum.xrf', 'bubblegum.xrf.model',
               'bubblegum.xrf.view'],
 )
+
+import shutil
+import os
+
+try:
+    os.mkdir(os.path.join(os.path.expanduser('~'), '.bubblegum'))
```

Comment:
This is the commit that really deals with rearranging the defaults.  https://github.com/licode/bubblegum/commit/5386ec643988e6561c2472d76e2ac2876a2d87be

How would you prefer this be done in the context of that commit